S7.Net/Conversion.cs

```diff
@@ -151,6 +151,51 @@ public static bool SelectBit(this byte data, int bitPosition)
             return (result != 0);
         }
 
+        /// <summary>
+        /// Helper to set a bit value to the given byte at the bit index.
+        /// </summary>
+        /// <param name="data"></param>
+        /// <param name="bitPosition"></param>
+        /// <param name="value"></param>
+        public static void SetBit(this ref byte data, int bitPosition, bool value)
+        {
+            if (bitPosition < 0 || bitPosition > 7)
```

Comment:
```suggestion
            if ((uint)bitPosition > 7)
```
With the `uint` cast any negative numbers will be interpreted as > int.MaxValue, thus > 7 and you can save one comparison / branch in the machine code.